src/notify.js

```diff
@@ -37,6 +39,8 @@ function Notify(title, options = {}) {
     this.permission = null;
     this.closeOnClick = closeOnClick;
     this.timeout = timeout;
+    this.swUrl = swUrl;
+    this.swRegistration = swRegistration;
```

Comment:
As suggested, I think this use method is likely not one most people will seek to use. I think we should remove this for the time being, and concentrate on a solid implementation of triggering notifications from within a SW. We can always add it later if there is a demand for it.